src/docs/automation-and-resiliency/namespace-recovery-and-responsibilities.md

```diff
@@ -0,0 +1,91 @@
+
+## What To Backup
+
+What would happen if some catastrophic failure completely destroyed one of the BC Government's OpenShift clusters? This is obviously not a very likely scenario, but it serves as a good thought experiment to consider what teams should think about backing up and knowing how to recover.
+
+If something like this were to happen, the Platform Team and our friends at DXC would work together to stand up a new cluster, and would redeploy all platform services, including the Product Registry. We would end up with a cluster with complete recovered services - including Vault and Artifactory - and a cluster full of *empty* namespaces with fresh access granted according to the Product Registry's listed Product Owner and Team Lead.
+
+This means it is the responsibility of the team to be able to recover *every* object and all data within your namespaces.
+
+## Why Backups are Important
+
+Obviously, it's not very likely that something will completely destroy an entire cluster. It might be easy to think about something like that and decide that, since it's so unlikely, it's not worth backing anything up - after all, what are the chances that anything on that scale might happen?
+
+But you don't need something of that scale in order to need backups.
+
+What if your new hire makes a mistake on their first day and deletes an important secret by accident? Would it be recoverable? How much of a problem would that pose for your team and application?
+
+Teams should be able to recover their applications gracefully from unforeseen issues and mistakes like this. It's not always possible to cover every possible issue, but the risks can be mitigated if you follow best practices on what to backup and how to recover those objects.
+
+## Recovering Different Tpes of Objects
+
+### General Deployment Objects
+
+General deployment objects are objects that you can recreate from scratch without suffering data loss. This class of object includes all of your Deployments, StatefulSets, pods, containers, NetworkPolicies, Services, Routes, most ConfigMaps, and some Secrets. 
+
+In order to determine if a ConfigMap or Secret should be classed as a general deployment object, ask yourself how difficult it would be to recover your application if you lost the data stored within the ConfigMap or Secret. If it's not a big deal (either because all that data is stored in Github or because generating a new password is acceptable), then it's a general deployment object. If not, we'll talk about how to handle this in the next section.
+
+These objects should be stored as **infrastructure-as-code**. Infrastructure-as-code means that you have stored your infrastucture configuration (ie: all the configuration of the general deployment objects) as code in your application's code repository. You can, with one (or close to one) push of a button, redeploy every object required for your application with the correct configuration.
+
```

Comment:
Is there a team with publicly stored infrastructure as code that could be linked here as an example?

Reply:
I added a link to RedHat's documentation on IaC, hopefully that will help!